internal/pkg/application/transform/transformers.go

```diff
@@ -132,10 +152,88 @@ func AirQualityObserved(ctx context.Context, msg iotcore.MessageAccepted, cbClie
 	return nil
 }
 
-func Device(ctx context.Context, msg iotcore.MessageAccepted, cbClient client.ContextBrokerClient) error {
-	v, ok := msg.GetBool(measurements.Presence)
+func IndoorEnvironmentObserved(ctx context.Context, msg core.MessageAccepted, cbClient client.ContextBrokerClient) error {
+	properties := []entities.EntityDecoratorFunc{
+		entities.DefaultContext(),
+		Location(msg.Latitude(), msg.Longitude()),
+		DateObserved(msg.Timestamp),
+	}
+
+	/*
+		ObjectURN: urn:oma:lwm2m:ext:3303
```

Comment:
if illuminance exists as its own type, perhaps we ought to add it in here with this comment for the temp and airquality